### DISCUSSION TITLE
Not Responding when Interface is selected in Audio/Network Settings

### DISCUSSION THREAD

**Discussion by th3Kan0b** [jamulussoftware/jamulus#2523] [Bug Investigation]
**Describe the bug**
<Every time I select USB Focusrite ASIO from the drop down menu, Jamulus freezes and shows not responding.  Everything else works as normal until the interface is selected -->

**To Reproduce**

**Expected behavior**

**Screenshots**

**Operating system**

**Version of Jamulus**

**Additional context**

**Comment by pljones** [comment]
This sounds like you've got a local set up problem.  I've moved it over to discussions so other users can comment more generally.

**Reply by dcorson-ticino-com** [reply]
I would uninstall and then reinstall the ASIO driver as a first step.
But I don't have a Focusrite interface so I can't go into details.
I hope someone with a Focusrite interface will speak up.
